core/Pos/Crypto/AsBinary.hs

```diff
@@ -44,14 +44,14 @@ checkLenImpl action name expectedLen len
 
 #define Ser(B, Bytes, Name) \
   instance (Bi B, Bi (AsBinary B)) => AsBinaryClass B where {\
-    asBinary = AsBinary . checkLen "asBinary" Name Bytes . encode ;\
-    fromBinary = decodeFull . checkLen "fromBinary" Name Bytes . encode }; \
+    asBinary = AsBinary . checkLen "asBinary" Name Bytes . serialize' ;\
+    fromBinary = decodeFull . checkLen "fromBinary" Name Bytes . getAsBinary }; \
```

Comment:
See [here](https://github.com/adinapoli-iohk/cardano-sl/commit/66858c1b93c4083055d7d045ce8e5cb54c05deac) for the full story around this change.